packages/flutter_tools/lib/src/debug_adapters/flutter_test_adapter.dart

```diff
@@ -20,6 +20,8 @@ import 'mixins.dart';
 /// A DAP Debug Adapter for running and debugging Flutter tests.
 class FlutterTestDebugAdapter extends DartDebugAdapter<FlutterLaunchRequestArguments, FlutterAttachRequestArguments>
     with PidTracker, TestAdapter {
+  // TODO(goderbauer): Figure out why super parameters in this class break internal google tests.
```

Comment:
Is there an open issue for this that it can reference?

Reply:
There is! Let me find and add it.